extensions/blocks/podcast-player/style.scss

```diff
@@ -100,30 +100,40 @@ $player-float-background: $light-gray-200;
 		height: $cover-image-size;
 	}
 
-	.jetpack-podcast-player__title {
-		display: flex;
-		flex-direction: column;
-		margin: 0;
-		overflow: hidden;
-	}
-
-	.jetpack-podcast-player__current-track-title {
-		font-size: $track-title-font-size;
-		margin: 0 0 $track-title-b-margin;
-	}
-
-	.jetpack-podcast-player__podcast-title {
-		font-size: $podcast-title-font-size;
-		color: $text-color;
-		margin: 0;
-	}
-
-	a.jetpack-podcast-player__podcast-title {
-		text-decoration: none;
-
-		&:hover,
-		&:focus {
-			color: $text-color-hover;
+	// Increasing specificity on the header titles and links for more consistency across themes and editor canvas
+	.jetpack-podcast-player__header {
```

Comment:
I think specifying the tag here should do the trick, too:

```css
h2.jetpack-podcast-player__header {
...
}
```

IMO, it would be better than wrapping selectively like this.